includes/Story_Post_Type.php

```diff
@@ -132,6 +140,21 @@ public function register() {
 			]
 		);
 
+		$active_publisher_logo_id = absint( get_option( Settings::SETTING_NAME_ACTIVE_PUBLISHER_LOGO ) );
+
+		register_post_meta(
+			self::POST_TYPE_SLUG,
+			self::PUBLISHER_LOGO_META_KEY,
+			[
+				'sanitize_callback' => 'absint',
```

Comment:
Maybe we could add a validation callback, to check if the id provided is a valid attachment id.

Reply:
I thought about this too but it felt like very defensive programming. We don't do this on other places either (e.g. original video ID when optimizing videos)